runtime/j9vm/jvm.c

```diff
@@ -1817,6 +1818,33 @@ jint JNICALL JNI_CreateJavaVM(JavaVM **pvm, void **penv, void *vm_args) {
 #else /* CALL_BUNDLED_FUNCTIONS_DIRECTLY */
 			zipFuncs = (J9ZipFunctionTable*) J9_GetInterface(IF_ZIPSUP, &j9portLibrary, j9binBuffer);
 #endif /* CALL_BUNDLED_FUNCTIONS_DIRECTLY */
+#ifdef WIN32
+			{
+				BOOLEAN conversionSucceed = FALSE;
+				int32_t size = 0;
+			
+				PORT_ACCESS_FROM_PORT(&j9portLibrary);
+				/* specialArgs.executableJarPath is retrieved from JavaVMInitArgs.options[i].optionString
+				 * which was set by Java Launcher in system default code page encoding.
+				 */
+				size = j9str_convert(J9STR_CODE_WINDEFAULTACP, J9STR_CODE_MUTF8, specialArgs.executableJarPath, strlen(specialArgs.executableJarPath), NULL, 0);
```

Comment:
Can `specialArgs.executableJarPath` be NULL here?  It would be better to cache the `strlen` call rather than repeating it here and below

Reply:
`specialArgs.executableJarPath` can be `NULL` but current block is within https://github.com/eclipse/openj9/blob/5a3774c9e698f1e0661420efebc0d93dd6075d1e/runtime/j9vm/jvm.c#L1806 hence it can't be `NULL`.
Cached the `strlen()` call result.